From 25e53eaf9e766713bed499cfff6f43855cd35e61 Mon Sep 17 00:00:00 2001
From: Christian Stewart <christian@paral.in>
Date: Tue, 10 May 2022 06:21:46 -0700
Subject: [PATCH] ti-tspa: add link to vpdma-1ba in root

Signed-off-by: Christian Stewart <christian@paral.in>
---
 WHENCE | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WHENCE b/WHENCE
index eab2f22..6f08e79 100644
--- a/WHENCE
+++ b/WHENCE
@@ -3034,6 +3034,7 @@ Licence: Redistributable. See LICENCE.tda7706-firmware.txt for details.
 Driver: ti-vpe - Texas Instruments V4L2 driver for Video Processing Engine
 
 File: ti/vpdma-1b8.bin
+Link: vpdma-1b8.bin -> ti/vpdma-1b8.bin
 
 Licence: Redistributable. See LICENCE.ti-tspa for details.
 
-- 
2.35.1

