From 7379f5ff5bd8c3f8d67db1345f4bb95850d24cd6 Mon Sep 17 00:00:00 2001
From: Christian Stewart <christian@paral.in>
Date: Wed, 30 Nov 2022 16:08:25 -0800
Subject: [PATCH] makefile: remove Wno-date-time warning

Signed-off-by: Christian Stewart <christian@paral.in>
---
 Makefile | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 7d79fcd..4d1de64 100644
--- a/Makefile
+++ b/Makefile
@@ -22,11 +22,7 @@ EXTRA_CFLAGS += -Wno-implicit-fallthrough
 # gcc-12
 EXTRA_CFLAGS += -Wno-address
 EXTRA_CFLAGS += -Wframe-larger-than=1648
-
-GCC_VER_49 := $(shell echo `$(CC) -dumpversion | cut -f1-2 -d.` \>= 4.9 | bc )
-ifeq ($(GCC_VER_49),1)
-EXTRA_CFLAGS += -Wno-date-time	# Fix compile error && warning on gcc 4.9 and later
-endif
+EXTRA_CFLAGS += -Wno-error=date-time
 
 EXTRA_CFLAGS += -I$(src)/include
 
-- 
2.38.1

